reprospect/test/sass/instruction/move.py

```diff
@@ -0,0 +1,23 @@
+import typing
+
+import regex
+
+from reprospect.test.sass.instruction.instruction import PatternBuilder, PatternMatcher
+
+
+class MovMatcher(PatternMatcher):
```

Comment:
Another possibility would be not to introduce a MovMatcher and just rely on `OpcodeModsWithOperandsMatcher`. An argument would be that this one doesn't need special adaptations like some of the others.